composer.json

```diff
@@ -126,22 +128,22 @@
         "drupal/views_bulk_operations": "^3.6",
         "drupal/xmlsitemap": "~1.0",
         "ext-imagick": "*",
-        "su-soe/soe_basic": "^8.1.0",
-        "su-sws/jumpstart_ui": "~8.1.0",
-        "su-sws/nobots": "~8.2.0",
-        "su-sws/react_paragraphs": "~8.1.0",
-        "su-sws/stanford_basic": "~8.4.0",
-        "su-sws/stanford_date_formats": "~8.1.0",
-        "su-sws/stanford_events": "~8.1.0",
-        "su-sws/stanford_fields": "~8.1.0",
-        "su-sws/stanford_image_styles": "~8.1.0",
-        "su-sws/stanford_news": "~8.2.0",
-        "su-sws/stanford_notifications": "~8.1.0",
-        "su-sws/stanford_person": "~8.1.0",
-        "su-sws/stanford_profile_helper": "~8.1.0",
-        "su-sws/stanford_ssp": "~8.1.0",
-        "su-sws/stanford_text_editor": "~8.1.0",
-        "su-soe/soe_paragraphs": "~8.1.0"
+        "su-soe/soe_basic": "dev-8.x-1.x",
+        "su-sws/jumpstart_ui": "dev-8.x-1.x",
+        "su-sws/nobots": "dev-8.x-2.x",
+        "su-sws/react_paragraphs": "dev-8.x-2.x",
+        "su-sws/stanford_basic": "dev-8.x-4.x",
+        "su-sws/stanford_date_formats": "dev-8.x-1.x",
+        "su-sws/stanford_events": "dev-8.x-1.x",
+        "su-sws/stanford_fields": "dev-8.x-1.x",
+        "su-sws/stanford_image_styles": "dev-8.x-1.x",
+        "su-sws/stanford_news": "dev-8.x-2.x",
+        "su-sws/stanford_notifications": "dev-8.x-1.x",
+        "su-sws/stanford_person": "dev-8.x-1.x",
+        "su-sws/stanford_profile_helper": "dev-8.x-1.x",
+        "su-sws/stanford_ssp": "dev-8.x-1.x",
+        "su-sws/stanford_text_editor": "dev-8.x-1.x",
+        "su-soe/soe_paragraphs": "dev-8.x-1.x"
```

Comment:
gotta fix these

Reply:
fixed, and squashed back into the previous commit. :)

Reply:
![nice](https://media1.tenor.com/images/3a4c31075aa8ad3cab6cec4863ed3018/tenor.gif?itemid=4480188)